### ISSUE TITLE
Get errors while running the code and need related code for training

### ISSUE THREAD

**Issue by Adnan1986** [zh460045050/LNSNet#10]
Hello,

Thank you for sharing your code. I am trying to run your algorithm on BSDS500 data sets and just need output labels and run-time so I could compare your work with mine and other algorithms.

I do appreciated if you could provide me with related training code.
my emails:
adnan.ghaderi@mdu.se
or
adnan.ghadri@gmail.com

While I wanted to run the demo I experienced following errors, Is there a way to fix it? 

`
../Superpixel/LNSNet-main/libs/layers/grm.py:118: UserWarning: volatile was removed and now has no effect. Use with torch.no_grad():  instead.
  self.Lambda = torch.autograd.Variable(torch.zeros(20 * 1 * 1).type(torch.float32), volatile=True)
/home/adnan/.local/lib/python3.8/site-packages/torch/functional.py:504: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at ../aten/src/ATen/native/TensorShape.cpp:3190.)
  return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]
[W NNPACK.cpp:53] Could not initialize NNPACK! Reason: Unsupported hardware.

`

Just in case, to install requirement on ubuntu 20.04 I install following library:
pip3 install torch torchvision
pip3 install tensorboardX

Thank you for your time,
Adnan
